engine/schema/src/main/java/com/cloud/upgrade/dao/Upgrade41400to41500.java

```diff
@@ -235,6 +238,267 @@ private void updateSystemVmTemplates(final Connection conn) {
         LOG.debug("Updating System Vm Template IDs Complete");
     }
 
+    private void addRolePermissionsForNewReadOnlyAndSupportRoles(final Connection conn) {
+        addRolePermissionsForReadOnlyAdmin(conn);
+        addRolePermissionsForReadOnlyUser(conn);
+        addRolePermissionsForAdminSupport(conn);
+        addRolePermissionsForUserSupport(conn);
```

Comment:
Minor nit - can we call the role 'SupportAdmin' and 'SupportUser'?

Reply:
> Minor nit - can we call the role 'SupportAdmin' and 'SupportUser'?

Updated